### DISCUSSION TITLE
Burn away mass calculation

### DISCUSSION THREAD

**Discussion by R-Godakandage** [firemodels/fds#13129] [Q&A]
Hi, 
I'm using fds 6.7.6. When we don't use the bulk density,  how's the mass within the cell is calculated? is it based on the material density and the thickness of the SURF ? ( I'm considering a 1 cell thick obstruction with a single layer of material). Thanks in advance.

**Comment by mcgratta** [comment]
Yes, but you should try a simple test case to verify.
